### PR TITLE
URL in bookmarks and history list not visible because they are have the same color (white) as the background #1726

### LINKED ISSUE #1726: URL in bookmarks and history list not visible because they are have the same color (white) as the background
URL is visible when user taps on the cell:

Bookmarks | History | Output
-|-|-
||

### PR DESCRIPTION
**Pull request by nimitparekh2020**
This issue fixed #1726
- Change URL colour

Bookmarks | History | Output
-|-|-
||

### CODE REVIEW

File: AlphaWallet/Browser/ViewModel/BrowserHistoryCellViewModel.swift [hunk 1]
```diff
@@ -39,7 +39,7 @@ struct BrowserHistoryCellViewModel {
     }
 
     var urlColor: UIColor? {
-        return Colors.appBackground
+        return UIColor(red: 0, green: 127, blue: 237)
```
Comment (hboon) [inline AlphaWallet/Browser/ViewModel/BrowserHistoryCellViewModel.swift:42]: Great. Can you:

1. Do it for bookmarks too?
2. `return Colors.appTint` instead, which is the same color, which is basically R.color.azure(). Using `Colors.appTint` where it's the app tint color helps us switch the tint color. This is used in the other repo too and that app has a different tint color.

Comment (nimitparekh2020) [reply AlphaWallet/Browser/ViewModel/BrowserHistoryCellViewModel.swift]: I make changes into the Bookmark and History section. @hboon kindly review that.

**Comment by nimitparekh2020** [comment]
@hboon I will keep in mind and doing squash commit.

**Comment by hboon** [comment]
@nimitparekh2020 I mean the commit message :)

**Comment by hboon** [comment]
@nimitparekh2020 good, but can you make it one line without the manual word wrap?

PS: This seems trivial, but I hope that ones we get past this and do it consistently, it'll all be good and helpful in the long run.

**Comment by nimitparekh2020** [comment]
> @nimitparekh2020 good, but can you make it one line without the manual word wrap?
> 
> PS: This seems trivial, but I hope that once we get past this and do it consistently, it'll all be good and helpful in the long run.

@hboon I checked Github there is limit of characters for word wrap that's why trim the line.
I tried but it didn't align into single line.

**Comment by hboon** [comment]
@nimitparekh2020 you are right, it's just in the GitHub UI. It's already a 1-liner in the commit itself. I'll merge this soon. Thanks!

**Pull request merged**